app/controllers/downvotes_controller.rb

```diff
@@ -0,0 +1,11 @@
+class DownvotesController < ApplicationController
+  def create
+    if current_user
+      @review = Review.find(params[:review_id])
+      @vote = @review.votes.find_or_create_by(user: current_user)
```

Comment:
In our UpvotesController and DownvotesController, we are combining creation of the flash notice and calling the increment or decrement method. in line 6 of the downvotes_controller, we have:

      redirect_to restaurant_path(@review.restaurant), notice: @vote.decrement

assigning that notice calls the decrement method on @vote, and the return of the decrement method is what we want to flash on the view